build.gradle

```diff
@@ -43,9 +43,9 @@ sonarqube {
 
 allprojects {
     repositories {
-        jcenter()
-        google()
         mavenCentral()
+        google()
+        jcenter()
```

Comment:
Should we not be removing Jcenter? Do we need it for anything ?

Reply:
There are couple of libraries we haven't updated to versions shipped in MavenCentral. I'm reluctant to make those changes here but we can schedule them @Peter-John-paystack

Reply:
@Peter-John-paystack we're moving away from GSON because the new data access classes are all in Kotlin.
Because of Java's type erasure, GSON cannot resolve the return types for the retrofit service functions and it makes the service calls fail.

Reply:
Cool happy with the Jcenter, but why Moshi and not Kotlin serialisable just out of interest?

Reply:
It caused quite a number of conflicts with the kotlin version that was already in the project.

Reply:
Ok cool cool, lets just make sure that we are going to do regression testing on all functionality if we are making this change please. 😊